### PR TITLE
chore: adjust copy around org level billing

### PR DESCRIPTION
**Pull request by kevcodez**
* Bunch of copy adjustments (not final)
* Fixed links to infrastructure page

### CODE REVIEW

File: packages/shared-data/plans.ts [hunk 1]
```diff
@@ -48,11 +51,27 @@ export const plans: PricingInformation[] = [
...
     nameBadge: '',
     costUnit: '/ month / project',
+    costUnitOrg: '/ month',
     href: 'https://supabase.com/dashboard/new/new-project',
     priceLabel: 'From',
     warning: 'Usage-based plan',
     priceMonthly: 25,
     description: 'For production applications with the option to scale.',
+    featuresOrg: [
+      '$10 Compute Credits',
```
Comment (joshenlim) [inline packages/shared-data/plans.ts:61]: should we keep the casing consistent? same for the other lines below too ("Realtime" vs "realtime" etc)
```suggestion
      '$10 compute credits',
```

File: studio/components/interfaces/Organization/BillingSettingsV2/TaxID/TaxID.tsx [hunk 2]
```diff
@@ -128,8 +128,9 @@ const TaxID = () => {
         <div className="sticky space-y-6 top-16">
           <div>
             <p className="text-base">Tax ID</p>
-            <p className="text-sm text-scale-1000">
-              If you would like to include specific tax ID(s) to your invoices.
+            <p className="text-sm text-scale-1000 pr-4">
+              If you would like to include specific tax ID(s) to your invoices. Old invoices are not
```
Comment (joshenlim) [inline studio/components/interfaces/Organization/BillingSettingsV2/TaxID/TaxID.tsx:132]: just to keep the same language as payment methods
```suggestion
              If you would like to include specific tax ID(s) to your invoices. Past invoices are not
```

File: studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx [hunk 3]
```diff
@@ -287,14 +289,47 @@ const ComputeInstanceSidePanel = () => {
...
+                                    <Tooltip.Arrow className="radix-tooltip-arrow" />
+                                    <div
+                                      className={[
+                                        'rounded bg-scale-100 py-1 px-2 leading-none shadow',
+                                        'border border-scale-200',
+                                      ].join(' ')}
+                                    >
+                                      <div className="flex items-center space-x-1">
+                                        <p className="text-scale-1200 text-sm">
+                                          ${Number(option.price * 672).toFixed(0)} - $
```
Comment (joshenlim) [inline studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx:324]: just curious - what do 672 and 744 represent?

**Review by joshenlim — APPROVED** [review]
just some minor comments but generally lgtm!

File: studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx (hunk 3 above)
Comment (kevcodez) [reply studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx:324]: 672 hours = 28 days
744 hours = 31 days

**Pull request merged**